src/components/create-user.html

```diff
@@ -1,95 +1,157 @@
-                        <label for="email">Email</label>
-                        <input type="text" id="email" name="email" placeholder="Your email..">
-                        <div>
-                            <button class="button">Create</button>
-                        </div>
-                    </form>
-                </div>
-            </div>
+  <template>
+    <style>
+      :host {
+        display: block
+      }
+
+      .div-center {
+        text-align: center;
+      }
+
+      input[type=text],
+      select {
+        width: 100%;
+        padding: 12px 20px;
+        margin: 8px 0;
+        display: inline-block;
+        border: 1px solid #87CEFA;
+        border-radius: 4px;
+        box-sizing: border-box;
+      }
+
+      div {
+        border-radius: 5px;
+        background-color: #f2f2f2;
+        padding: 20px;
+      }
+
+      .border {
+        border: #666 1px solid;
+        width: 20%;
+        margin: auto;
+        margin-top: 25px;
+        padding-bottom: 25px;
+      }
+
+      .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+    </style>
+
+    <h1 class="div-center">Create User</h1>
+    <div>
+      <div class="container">
+        <div class="form-border">
+          <form>
+            <label for="fname">First Name</label>
+            <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+            <label for="lname">Last Name</label>
+            <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+            <label for="phone">Phone</label>
+            <input type="text" id="phone" name="phone" placeholder="Your phone number..">
+
+            <label for="email">Email</label>
+            <input type="text" id="email" name="email" placeholder="Your email..">
+          </form>
+          <div>
+            <button class="button" on-click="createUser">Create User</button>
+            <button class="button" on-click="deleteAllUsers">Delete All Users</button>
+          </div>
         </div>
-    </template>
+      </div>
+    </div>
+  </template>
+
+  <script type="module">
+    import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
+    
+    function generateId() {
+      return Math.floor(1000 + Math.random() * 9000);
+    }
+
+    let localStorage = new LocalStorage(
+        generateId(),
+        'Ted'
+    );
+  
+    class CreateUser extends PolymerElement {
+      static get is() { return 'create-user'; }
+      constructor() {
+        super();
+      }
+      
+      createUser(user) {
+        localStorage.createUser(localStorage);
```

Comment:
you'll want to keep all `localStorage` work from within your `LocalStorage` module. Or, more directly, you shouldn't be using `localStorage` in your components.

Reply:
Are we talking about the passing of localStorage in the calls or instantiating it and using the functions like that all together within the component?

Reply:
What I was doing to test my functions from LocalStorage was importing it to my component, making an instance of it (to be able to use it's methods), and then calling them like that.

Reply:
>  Is this the wrong way to go about importing and using the Class you're importing?

Nope, not at all. I just want your components to actually use your `LocalStorage` class to interface with `localStorage`, not `localStorage` directly. So, here in `create-user.html`, you'd call something like `LocalStorage.createUser(user)` once you've gotten all the user information data together into an object (or something). Your `LocalStorage` class then would do all the work to get it into the list of users and update `localStorage` so that the create user process is "saved".

Reply:
> Are we talking about the passing of localStorage in the calls...

I wasn't initially but you certainly won't want to be doing that.

> ... or instantiating it and using the functions like that all together within the component?

Yeah, you'll still want to import it and create an instance of the class for your app to use.

Reply:
Made some changes to create-user.html and how the createUser function is being utilized and how to user object being passed into that function is being built. Can you take a look and see if these changes are valid according to what we were discussing previously ^?

Reply:
Or should I set the properties like this and pass those props into the function calls?

![image](https://user-images.githubusercontent.com/51921679/61394666-3da94480-a889-11e9-8b0c-9fa14600fa80.png)

Reply:
This is just an example from the docs I'm looking through. I'd modify it to my needs for the project of course.

Reply:
I _think_ the `createUser` call is in much better shape now. You'll still need to remove any direct references to `localStorage` from the component (the only thing we want to access `localStorage` directly is the API).

Reply:
Ok great, I didn't do any refactoring to anything but the createUser(), wanted to make sure I did it right there before moving on to the rest of the app. Thanks again for the help!